### DISCUSSION TITLE
Missing Critical Features, Quirky

### DISCUSSION THREAD

**Discussion by burtondev** [solidusio/solidus#5393] [New Admin UI/UX]
I'm new to Solidus so IDK if I'm overlooking something (if I am, please forgive me), but based solely on your Admin panel demo, there's critical and high priority features missing.

I discovered Solidus while seeking an alternative to Shopify's limited variant options. Solidus looks promising in that regard, but the Admin panel kinda scares me. There's no Customers section to see a list of all customers, their locations, emails, etc. I don't see any way to cancel, refund, or process orders (duplicate, archive, automate, etc.) I don't see a way to batch create or import products (e.g. via CSV file). These functions I consider critical.

Also there's no sales analytics or reporting. Shopify provides detailed reporting for total sales, sales by channel, conversion rate, AOV and more. While arguably not critical, this should be highly prioritized, as it's hard to run a business for any length of time successfully without these types of metrics.

Finally, I found the demo a bit quirky, for example getting "Loading failed" messages, or spontaneously logged and locked out and had to wait a while before logging back in. I couldn't find a way to add a Taxonomy in either the new or legacy admin. Last night I was able to add new option types, but today the create button is missing?

I'd love to find a solution at long last for products with a high amount of variants, and Solidus looks promising. Are there any plans to develop and release these critical/high priority features for the Admin? And are these quirks only in the demo or should I expect as much if I try to implement this platform?

**Comment by jarednorman** [comment]
The demo doesn't make all functionality available. Functions that modify data aren't available, which is why you can't see things like the refund and cancel buttons. I'm not sure why the customer management isn't accessible, but that appears to be a product of how the permissions of the demo user were set up. I assure you there is customer management functionality. The new admin will enable some bulk actions, but bulk imports of products can be done with the solidus_importer extension. The new admin is under heavy active development right now.

If you want to properly evaluate Solidus, I recommend spinning up a store locally and trying it out.

**Comment by burtondev** [comment]
Hello, @jarednorman! Thanks for your reply. I'm still familiarizing myself with the documentation, so thanks for pointing me to the solidus_importer extension (for anyone else reading this with the same question, you can find it [here](https://github.com/solidusio-contrib/solidus_importer) - it's one of the [Solidus Community Extensions](https://github.com/solidusio-contrib)). 

I look forward to trying it out soon! I'll leave this discussion open for a little bit in case I have any related questions after testing it out.
